### PR TITLE
Feature/24771 CVE scanning

### PR DESCRIPTION
**Pull request by RemcoRitense**
Added java dependency vulnerability scan.

### CODE REVIEW

File: .java-version
Comment (Hildebrand-Ritense) [inline .java-version:1]: Personally don't like this. It complicates version matching with what version you have specifically running locally. Also, this is out of scope for this PR. I do know others like it, because it's a more precise match, but I've never seen bugs coming from a specific Java minor. I'm going to leave this to the Valtimo team...

File: gradle/build.gradle
Comment (Hildebrand-Ritense) [inline gradle/build.gradle:1]: in what scenario did you need this? I don't think this was needed for the other (license-report) module.

File: gradle/cve-report/cve-exclusions.xml
Comment (Hildebrand-Ritense) [inline gradle/cve-report/cve-exclusions.xml:1]: Can you explain why we're excluding guava? Good to have this documented :)

Comment (Hildebrand-Ritense) [inline gradle/cve-report/cve-exclusions.xml:1]: let's rework the contents a bit so it's clear it's an example / placeholder and that it needs the configuration in the gradle/cve-report/build.gradle file for it to do anything.

**Review by Hildebrand-Ritense — COMMENTED** [review]
almost there 👍

**Review by Hildebrand-Ritense — APPROVED** [review]
lgtm

File: gradle/build.gradle [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tasks.withType(PublishToMavenRepository) {
+    enabled = false
+}
+tasks.withType(PublishToMavenLocal) {
```
Comment (marijnritense) [inline gradle/build.gradle:4]: Why is this being disabled?

Comment (RemcoRitense) [reply gradle/build.gradle:4]: Let me look into this, this piece of code was found in multiple locations and I might have missed this one.

Comment (Hildebrand-Ritense) [reply gradle/build.gradle:4]: I think I added this way back because the gradle folder, being a folder/module with submodules, shouldn't itself be published. I asked Remco to double check this (see my comment 2 days ago) since this was added way back.

Comment (marijnritense) [reply gradle/build.gradle:4]: Okay, that makes sense!
